client/js/components/Button/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import {Text, TouchableOpacity} from 'react-native'
+import styles from './Button.styles'
+
+const Button = ({children}) => {
+  return (
+    <>
+      <TouchableOpacity style={styles.button}>
+        <Text style={styles.font}>{children}</Text>
+      </TouchableOpacity>
+    </>
+  )
+}
```

Comment:
Could default to writing your 'fat-arrow functions' with an implicit return and avoid wrapping elements in fragments if possible

```suggestion
const Button = ({children}) => (
  <TouchableOpacity style={styles.button}>
    <Text style={styles.font}>{children}</Text>
  </TouchableOpacity>
)
```